packages/core/src/domain/configuration/experimentalFeatures.spec.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  updateExperimentalFeatures,
+  isExperimentalFeatureEnabled,
+  resetExperimentalFeatures,
+} from './experimentalFeatures'
+
+describe('experimentalFeatures', () => {
+  it('initial state is empty', () => {
+    resetExperimentalFeatures()
+    expect(isExperimentalFeatureEnabled(undefined)).toBeFalse()
+    expect(isExperimentalFeatureEnabled('foo')).toBeFalse()
+    expect(isExperimentalFeatureEnabled('bar')).toBeFalse()
+    expect(isExperimentalFeatureEnabled('')).toBeFalse()
+  })
+
+  it('updateExperimentalFeatures', () => {
+    updateExperimentalFeatures(['foo'])
```

Comment:
it would be preferable to run a `resetExperimentalFeatures` in an `afterEach` block in order to not leak features between tests